frontend/workflows/projectCatalog/src/details/helpers.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import type { ReactTimeagoProps as TimeAgoProps } from "react-timeago";
+import TimeAgo from "react-timeago";
+import { Grid, Link, styled, Typography } from "@clutch-sh/core";
+import { faClock } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const StyledLink = styled(Link)({
+  whiteSpace: "nowrap",
+});
+
+const unitformatter = (unit: string): string => {
```

Comment:
Should unit have stronger typings?

Reply:
Probably, this was copied from internal and I need to export it into a reusable component. I have a ticket to track it and I'll make sure to type it better, need to make sure I understand all the use cases for it first.